plugin.video.crackle/.gitattributes

```diff
@@ -0,0 +1,17 @@
+# Auto detect text files and perform LF normalization
```

Comment:
please drop from submission. If you're automating submission try to add to .gitatttributes